test/e2e/storage/vsphere/vsphere_volume_master_restart.go

```diff
@@ -167,13 +167,13 @@ var _ = utils.SIGDescribe("Volume Attach Verify [Feature:vsphere][Serial][Disrup
 			expectVolumeToBeAttached(nodeName, volumePath)
 		}
 
-		ginkgo.By("Restarting kubelet on master node")
-		masterAddress := framework.GetMasterHost() + ":22"
-		err := restartKubelet(masterAddress)
-		framework.ExpectNoError(err, "Unable to restart kubelet on master node")
+		ginkgo.By("Restarting kubelet on instance node")
+		instanceAddress := framework.APIAddress() + ":22"
+		err := restartKubelet(instanceAddress)
+		framework.ExpectNoError(err, "Unable to restart kubelet on instance node")
 
```

Comment:
instance of what?
- instance node -> control plane instance? control plane node?
- instanceAddress -> controlPlaneInstanceAddress? cpInstanceAddress?